pkg/remote/remote.go

```diff
@@ -45,6 +51,27 @@ type Remote interface {
 	// and returns the required data in a standard format.
 	Hook(r *http.Request) (*common.Hook, error)
 
+	// Oauth2Transport
+	Oauth2Transport(r *http.Request) *oauth2.Transport
+
+	// GetOrgs returns all allowed organizations for remote.
+	GetOrgs() []string
+
+	// GetOpen returns boolean field with enabled or disabled
+	// registration.
+	GetOpen() bool
+
 	// Default scope for remote
 	Scope() string
 }
+
+func New(conf *config.Config) (Remote, error) {
```

Comment:
It seems not to have switch other driver info...
How can we get `DRONE_REMOTE_DRIVER`'s Remote struct? :sweat_drops: